build-parent/pom.xml

```diff
@@ -282,6 +284,11 @@
                 <version>${project.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>org.assertj</groupId>
+                <artifactId>assertj-core</artifactId>
+                <version>${assertj.version}</version>
```

Comment:
This needs to be declared in test scope

Reply:
Well, it's only in `dependencyManagement` and I usually dont' use scopes in depMgmt but since other deps here do use test scope, I will of course add it. Again, thanks for pointing that out.